backend/graphene_form/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+
+
+class FormWithContext(forms.Form):
+    def __init__(self, *args, **kwargs):  # pylint: disable=return-in-init
+        self._context = kwargs.pop('context', None)
+        return super().__init__(*args, **kwargs)
+
+    @property
+    def context(self):
+        if not self._context:
+            raise ValueError('Make sure you pass the context when instancing the Form')
```

Comment:
I wanted to keep `FormContext` flexible enough to be used even without context. I know it might not make sense when the name of the class has `Context` in it, but 😄